docs/en/writing-running-appium/tutorial/swipe/android-layout-direction.md

```diff
@@ -0,0 +1,21 @@
+## Android 'UIScrollable' swipe: Add scroll layout
+
+ScrollView can be horizontal and vertical. When UIAutomator fails
```

Comment:
There are horizontal or vertical scroll views

Reply:
If UIAutomator fails to automatically perform the scroll then explicitly specifying the layout could solve the issue.